channels/chan_simpleusb.c

```diff
@@ -329,6 +329,8 @@ struct chan_simpleusb_pvt {
 
 	struct rxaudiostatistics rxaudiostats;
 	
+	int legacyaudioscaling;
```

Comment:
If this is only ever 1 or 0, this would be more semantic to be an `unsigned int:1` (1 bit instead of an int).

Reply:
Hopefully this is only a temporary cfg that will be deleted at some point, and thus having its semantics be as nice as possible might actually discourage its later deletion.